### ISSUE TITLE
Sorted set processed in reverse

### ISSUE THREAD

**Issue by fristedt** [sripathikrishnan/redis-rdb-tools#124]
It appears that the `zadd` callback is called in reverse order (descending, highest to lowest score). Is there any way to process the sorted set in ascending order?

**Comment by sripathikrishnan** [comment]
No, there isn't a way to control that. We process the entries in the order in which they are *stored* in the rdb file. The parser is stateless, it is designed to use as little memory as possible during parsing. Returning records in a different order will require holding all elements in memory, and that isn't a good idea.

You can easily solve this by writing a new class that extends `RdbCallback`. Accumulate records in `zadd`, in `end_sorted_set` method sort all the records and apply your business logic.
